### PR TITLE
IGNITE-15263 Fix QuerySqlField.alias for affinity field

### PR DESCRIPTION
**Pull request by timoninmaxim**
Now affinity key correctly handles alias fields.

### CODE REVIEW

File: modules/core/src/main/java/org/apache/ignite/internal/processors/query/QueryUtils.java [hunk 1]
```diff
@@ -557,6 +557,9 @@ public static QueryTypeCandidate typeForQueryEntity(GridKernalContext ctx, Strin
                         if (!F.isEmpty(qryEntity.getKeyFields()) && qryEntity.getKeyFields().contains(affField0)) {
                             affField = affField0;
 
+                            if (desc.aliases().containsKey(affField))
+                                affField = desc.aliases().get(affField);
```
Comment (AMashenkov) [inline modules/core/src/main/java/org/apache/ignite/internal/processors/query/QueryUtils.java:561]:
```suggestion
                            affField = desc.aliases().getOrDefault(affField, affField);
```

File: modules/core/src/main/java/org/apache/ignite/internal/processors/query/QueryUtils.java [hunk 2]
```diff
@@ -578,6 +581,9 @@ public static QueryTypeCandidate typeForQueryEntity(GridKernalContext ctx, Strin
                     ((GridCacheDefaultAffinityKeyMapper)keyMapper).affinityKeyPropertyName(desc.keyClass());
 
                 if (affField != null) {
+                    if (desc.aliases().containsKey(affField))
+                        affField = desc.aliases().get(affField);
```
Comment (AMashenkov) [inline modules/core/src/main/java/org/apache/ignite/internal/processors/query/QueryUtils.java:585]:
```suggestion
                    affField = desc.aliases().getOrDefault(affField, affField);
```

Comment (timoninmaxim) [reply modules/core/src/main/java/org/apache/ignite/internal/processors/query/QueryUtils.java]: @AMashenkov , hi! Thanks for review. Applied both suggestions with an even additional improvement.

**Review by ivandasch — APPROVED** [review]
Looks good to me

**Pull request closed without merging**